interactions/client.py

```diff
@@ -579,8 +586,34 @@ def decorator(coro: Coroutine) -> Any:
 
         return decorator
 
+    @staticmethod
+    def _find_command(commands: List, command: str) -> ApplicationCommand:
+        """
+        Iterates over commands and returns an ApplicationCommand if it matches the name from command
+
+        :ivar commands: The list of dicts to iterate through
+        :type commands: List[ApplicationCommand]
+        :ivar command: The name of the command to match:
+        :type command: str
+        """
```

Comment:
Add a `:return` and `:rtype` to documentation that states that this will return an `ApplicationCommand`